TACC/Core-Portal
# Bug/FP-1011 - Fix large image preview loading artifact

## Overview: ##
Fix large image preview loading artifact

## Related Jira tickets: ##

https://jira.tacc.utexas.edu/browse/FP-1011
https://jira.tacc.utexas.edu/browse/FP-1200

## Summary of Changes: ##
Fixed logics for spinner is showing concurrently with an image.
Also, the image preview not showing (FP-1200) was resolved.

## Testing Steps: ##
1. Go to My account
2. Go to My Data
3. Open any large image (more than 5mb) for preview
4. Check if there is any image concurrent with the spinner.

## UI Photos:

https://user-images.githubusercontent.com/34716439/132888323-e372274a-04cf-43c2-9a20-088a4b3485bd.mov


## Notes: ##
